CDS/scripts/dailymotion_upload.pl

```diff
@@ -477,9 +472,9 @@ sub is_imageurl_valid {
 	language	 => $videolanguage,
 	access_token => $server->{'access_token'}
 };
-			  
-if (is_imageurl_valid($capi->{'response'}->{'media'}->{'data'}->{'media'}->{'trailImage'}->{'master'}->{'file'})) {
-	$content->{'thumbnail_url'} = $capi->{'response'}->{'media'}->{'data'}->{'media'}->{'trailImage'}->{'master'}->{'file'};
+
+if (is_imageurl_valid($imageurl)) {
```

Comment:
where does `$imageurl` come from? can't see it defined in the diff

Reply:
That is the URL for the holding image. This is already being input to the script but not used.

Reply:
The tag for it is <video_holding_image> which is in the diff. It is loaded into $imageurl on line 59.